tests/js/searchtools.js

```diff
@@ -31,6 +31,36 @@ describe('Basic html theme search', function() {
 
 });
 
+describe("htmlToText", function() {
+  it("basic case", () => {
+    expect(Search.htmlToText(`
+    <html>
+    <div class="body" role="main">
+      <section id="getting-started">
+        <h1>Getting Started</h1>
+        <p>Some text</p>
+      </section>
+    </div>
+    </html>`).trim().split(/\s+/)).toEqual(['Getting', 'Started', 'Some', 'text']);
+  });
+
+  it("will start reading from the anchor", () => {
+    expect(Search.htmlToText(`
+    <html>
+    <div class="body" role="main">
+      <section id="getting-started">
+        <h1>Getting Started</h1>
+        <p>Some text</p>
+      </section>
+      <section id="other-section">
+        <h1>Other Section</h1>
+        <p>Other text</p>
+      </section>
+    </div>
+    </html>`, '#other-section').trim().split(/\s+/)).toEqual(['Other', 'Section', 'Other', 'text']);
```

Comment:
Could you use this single (larger) HTML content as the input for both test cases?  I think that'll make it clearer that selecting a summary with-or-without an anchor produces different results (in fact, ideally we might want to test without an anchor, with one anchor, and then with a second anchor -- and confirm that cases two and three produce different summaries).